packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -197,7 +197,7 @@ class AbstractPouchDB extends EventEmitter {
         callback = opts;
         opts = {};
       }
-      if (typeof doc !== 'object' || Array.isArray(doc)) {
+      if (doc == null || typeof doc !== 'object' || Array.isArray(doc)) {
```

Comment:
Checking `null` non-strict will match every falsy value, you can reduce it to just `!doc`
```suggestion
      if (!doc || typeof doc !== 'object' || Array.isArray(doc)) {
```

Alt.:
```suggestion
      if (doc === null || typeof doc !== 'object' || Array.isArray(doc)) {
```

Reply:
> Checking `null` non-strictly will match every _falsy_ value, you can reduce it to just `!doc`

Checking `null` non-strictly will match `undefined` and `null` only.

```js
> [0,'',false,null,undefined].map(it => it == null)
[ false, false, false, true, true ]
```

Reply:
Either suggestion will work though :+1: